hackathon_site/dashboard/frontend/src/pages/Teams/Teams.tsx

```diff
@@ -2,13 +2,43 @@ import React from "react";
 // import styles from "./Teams.module.scss";
 import Header from "components/general/Header/Header";
 import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import styles from "pages/Teams/Teams.module.scss";
+
+import TeamCardAdmin from "components/dashboard/TeamCardAdmin/TeamCardAdmin";
+import InventorySearch from "components/inventory/InventorySearch/InventorySearch";
+import { teamsList } from "testing/mockData";
+
+// import TeamCardAdmin from "../../components/dashboard/TeamCardAdmin/TeamCardAdmin";
 
 const Teams = () => {
+    const CardComponents = teamsList.map((team) => (
+        <Grid
+            item
+            md={3}
+            sm={4}
+            xs={6}
+            className={styles.dashboardGridItem}
+            key={0}
+            data-testid="team"
+            direction="column"
+            grid-template-column="true"
```

Comment:
eventually, when a use clicks a team card, it will bring them to the team detail page, could you make a placeholder function on the grid item for this

e.g. `onClick={() => alert(`Goes to team-detail for team ${team.TeamName}`)}`

Reply:
added!